\documentclass[12pt,a4paper]{article}

\usepackage{fancyhdr}
\usepackage{graphicx}
\usepackage{placeins}
\usepackage{adjustbox}


\begin{document}

\pagestyle{fancy}
\fancyhf{}
\chead{Short summary report}

\begin{table}[t]
\centering
\caption {rnaQUAST metrics for assembled transcripts. In each row the best values are indicated with \textbf{bold}. For the transcript metrics (rows 4, 5, 6, 9, 13, 25, 26, 27) we highlighted the best \textbf{relative} values i.e. divided by the total number of transcripts in the corresponding assembly.}
\begin{adjustbox}{width=1\textwidth}
\small
\begin{tabular}{|l*{2}{|r}|}
\hline
\textbf{METRICS/TRANSCRIPTS}                            & \textbf{k25\_35\_45\_merged\_+100} \\ \hline\hline
\multicolumn{2}{l}{\bf DATABASE METRICS}                                                  \\ \hline
Genes                                                   & 46739                  \\
Avg. number of exons per isoform                        & 4.266                  \\ \hline
\multicolumn{2}{l}{\bf BASIC TRANSCRIPTS METRICS}                                         \\ \hline
Transcripts                                             & 41126                  \\
Transcripts $>$ 500 bp                                  & \textbf{18549}         \\
Transcripts $>$ 1000 bp                                 & \textbf{10913}         \\ \hline
\multicolumn{2}{l}{\bf ALIGNMENT METRICS}                                                 \\ \hline
Aligned                                                 & \textbf{41101}         \\
Uniquely aligned                                        & 39923                  \\
Multiply aligned                                        & 161                    \\
Unaligned                                               & \textbf{25}            \\ \hline
\multicolumn{2}{l}{\bf ALIGNMENT METRICS FOR NON-MISASSEMBLED TRANSCRIPTS}                \\ \hline
Avg. aligned fraction                                   & \textbf{0.991}         \\
Avg. alignment length                                   & \textbf{782.832}       \\
Avg. mismatches per transcript                          & \textbf{0.395}         \\ \hline
\multicolumn{2}{l}{\bf ALIGNMENT METRICS FOR MISASSEMBLED (CHIMERIC) TRANSCRIPTS}          \\ \hline
Misassemblies                                           & \textbf{0}             \\ \hline
\multicolumn{2}{l}{\bf ASSEMBLY COMPLETENESS (SENSITIVITY)}                               \\ \hline
Database coverage                                       & \textbf{0.357}         \\
50\%-assembled genes                                    & \textbf{9665}          \\
95\%-assembled genes                                    & \textbf{5824}          \\
50\%-covered genes                                      & \textbf{10844}         \\
95\%-covered genes                                      & \textbf{6354}          \\
50\%-assembled isoforms                                 & \textbf{10835}         \\
95\%-assembled isoforms                                 & \textbf{6194}          \\
50\%-covered isoforms                                   & \textbf{12193}         \\
95\%-covered isoforms                                   & \textbf{6767}          \\
Mean isoform coverage                                   & \textbf{0.761}         \\
Mean isoform assembly                                   & \textbf{0.699}         \\ \hline
\multicolumn{2}{l}{\bf ASSEMBLY SPECIFICITY}                                              \\ \hline
50\%-matched                                            & \textbf{38563}         \\
95\%-matched                                            & \textbf{32904}         \\
Unannotated                                             & \textbf{1092}          \\
Mean fraction of transcript matched                     & \textbf{0.934}         \\ \hline
\end{tabular}
\end{adjustbox}
\end{table}

\FloatBarrier
\clearpage
\lfoot{generated by rnaQUAST}

\begin{figure}[t]
\centering
\includegraphics[width = \linewidth]{/projects/btl/datasets/celegans/N2strain/RNA-Seq/RNA-QUAST/k25_35_45_merged_+100_output/transcript_length.png}
\caption{Plot showing cumulative transcript length distribution. Each point represents the number of transcripts in the assembly with the corresponding length or longer; black dashed line corresponds to the database isoforms; the plot is given in logarithmic scale.}
\end{figure}
\FloatBarrier
\clearpage


\begin{figure}[t]
\centering
\includegraphics[width = \linewidth]{/projects/btl/datasets/celegans/N2strain/RNA-Seq/RNA-QUAST/k25_35_45_merged_+100_output/mismatch_rate.png}
\caption{Plot showing cumulative substitution errors per alignment distribution. Each point represents the number of alignments with the corresponding number of mismatches or greater; the plot is given in logarithmic scale.}
\end{figure}
\FloatBarrier
\clearpage


\begin{figure}[t]
\centering
\includegraphics[width = \linewidth]{/projects/btl/datasets/celegans/N2strain/RNA-Seq/RNA-QUAST/k25_35_45_merged_+100_output/NAx.png}
\caption{Nx plot for transcripts. Nx is a maximal number $N$, such that the total length of all transcripts longer than $N$ bp is at least $x\%$ of the total length of all transcripts.}
\end{figure}
\FloatBarrier
\clearpage


\begin{figure}[t]
\centering
\includegraphics[width = \linewidth]{/projects/btl/datasets/celegans/N2strain/RNA-Seq/RNA-QUAST/k25_35_45_merged_+100_output/x-matched.png}
\caption{Plot showing cumulative transcript match histogram. Each bar represents the number of transcripts with matched fraction equal to or greater than the value on $x$ axis; transcript matched fraction is calculated as the number of its bases covering an isoform divided by the transcript length.}
\end{figure}
\FloatBarrier
\clearpage


\begin{figure}[t]
\centering
\includegraphics[width = \linewidth]{/projects/btl/datasets/celegans/N2strain/RNA-Seq/RNA-QUAST/k25_35_45_merged_+100_output/x-assembled.png}
\caption{Plot showing cumulative isoform assembly histogram. Each bar represents the number of isoforms with assembled fraction equal to or greater than the value on $x$ axis; isoform assembled fraction is calculated as the maximum number of captured by single assembled transcript bases divided by the total isoform length.}
\end{figure}
\FloatBarrier
\clearpage


\begin{figure}[t]
\centering
\includegraphics[width = \linewidth]{/projects/btl/datasets/celegans/N2strain/RNA-Seq/RNA-QUAST/k25_35_45_merged_+100_output/x-covered.png}
\caption{Plot showing cumulative isoform coverage histogram. Each bar represents the number of isoforms with covered fraction equal to or greater than the value on $x$ axis; isoform covered fraction is calculated as the number of covered bases (by all transcripts in the assembly) divided by the total isoform length.}
\end{figure}
\FloatBarrier
\clearpage


\end{document}
